generator/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM golang:1.9.2
```

Comment:
I would recommend `golang:latest` to pull in the latest stable, rather than a specific release.